### PR TITLE
rename knative/eventing-sources to knative/eventing-contrib

### PR DESCRIPTION
**Pull request by n3wscott**
Related to https://github.com/knative/eventing-contrib/issues/372

Leaves gcr.io links alone.

**NOTE:** gcr.io will change for 0.7 release.

```release-note
`knative/eventing-sources` changed to `knative/eventing-contrib`
```

### CODE REVIEW

**Comment by RichieEscarez** [comment]
Should these changes be Cherry picked into the 0.6 branch too?

**Comment by n3wscott** [comment]
No cherrypick for 0.6.

This is for current master and >=0.7.

**Comment by chaodaiG** [comment]
The e2e tests are broken in master branch currently, I'm looking into it right now. 

link: #1435

File: docs/install/Knative-with-any-k8s.md [hunk 1]
```diff
@@ -84,7 +84,7 @@ your Knative installation, see
    kubectl apply --filename https://github.com/knative/serving/releases/download/v0.6.0/serving.yaml --selector networking.knative.dev/certificate-provider!=cert-manager \
    --filename https://github.com/knative/build/releases/download/v0.6.0/build.yaml \
    --filename https://github.com/knative/eventing/releases/download/v0.6.0/release.yaml \
-   --filename https://github.com/knative/eventing-sources/releases/download/v0.6.0/eventing-sources.yaml \
+   --filename https://github.com/knative/eventing-contrib/releases/download/v0.6.0/eventing-sources.yaml \
```
Comment (samodell) [inline docs/install/Knative-with-any-k8s.md]: @n3wscott  -- just to confirm, when we rename the repo to contrib, are we still going to release an "eventing-sources.yaml" file, or will that be renamed as well? We can open an issue about updating this later in the latter is true, just to make sure we don't miss it.

**Comment by samodell** [comment]
Thanks for this, @n3wscott ! Looks like there are a few conflicts to resolve, and I added a comment with one question.

**Comment by n3wscott** [comment]
Just the repo name, not the release has been changed.

**Comment by n3wscott** [comment]
@samodell Updated.

**Comment by n3wscott** [comment]
ping, before more merge conflicts?

**Pull request merged**